public/video-ui/src/components/VideoEdit/formComponents/VideoPoster.js

```diff
@@ -19,7 +19,7 @@ class VideoPosterImageEdit extends React.Component {
 
   render () {
 
-    if (!this.props.editMode) {
+    if (this.props.video && !this.props.editMode) {
```

Comment:
[PropTypes](https://facebook.github.io/react/docs/typechecking-with-proptypes.html) will help here as we can enforce `this.props.video` is required.